map/migrations/0005_remove_mapentry_position.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.0.3 on 2018-03-19 21:40
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('map', '0004_mapentry_position'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='mapentry',
+            name='position',
+        ),
+    ]
```

Comment:
It seems that you are adding and removing fields in these migrations. This will cause data loss.. so not sure if it's what we want :)

Reply:
An okay, you can scrap all the migrations, right? I think they neutralize each other.